### PR TITLE
Frontend fix translations

### PR DESCRIPTION
**Pull request by thisconnect**
- [x] update locize if this change is approved.

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -1163,7 +1164,7 @@
     "label": "Your address",
     "onlyThisCoin": {
       "description": "To receive other tokens, enable them in the settings. If you deposit other tokens, they might not be accessible.",
-      "warning": "Make sure to only receive {{accountName}} on this address."
+      "warning": "Make sure to only receive {{coinName}} ({{coinUnit}}) on this address."
```
Comment (thisconnect) [inline frontends/web/src/locales/en/app.json:1167]: TODO: change in locize

File: frontends/web/src/routes/buy/info.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ class BuyInfo extends Component<Props, State> {
         if (options === undefined) {
             return <Spinner text={t('loading')} />;
         }
-        const name = (code && isBitcoin(code)) ? 'Bitcoin' : 'crypto';
+        const name = (code && isBitcoin(code)) ? 'Bitcoin' : t('buy.info.crypto');
```
Comment (thisconnect) [inline frontends/web/src/routes/buy/info.tsx:113]: @benma this is used as `{{name}}` in the translation, should I rename to `{{cryptoOrBitcoin}}` to make it more clear WDYT?

File: frontends/web/src/locales/en/app.json [hunk 3]
```diff
@@ -525,7 +526,7 @@
         "title": "Welcome to your one stop shop for buying {{name}}"
       },
       "next": "Next",
-      "selectPlaceholder": "Select a coin",
+      "selectLabel": "Select an account",
```
Comment (thisconnect) [inline frontends/web/src/locales/en/app.json:529]: Already updated in locize

Screen Shot 2021-05-21 at 12 49 02 PM

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (benma) [reply frontends/web/src/locales/en/app.json:1167]: imho you can drop coinUnit to spare translation cycles.

did you already change it in locize? make sure to do it in every language. let's do that first before merging.

File: frontends/web/src/routes/buy/info.tsx (hunk 2 above)
Comment (benma) [reply frontends/web/src/routes/buy/info.tsx:113]: keep, save to effort to change it in all languages

File: frontends/web/src/locales/en/app.json (hunk 3 above)
Comment (benma) [reply frontends/web/src/locales/en/app.json:529]: please don't edit text in the json file, only in locize: https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/docs/i18n.md

deleting the `selectPlaceholder` is good.

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/locales/en/app.json:1167]: I'll do it now.

**Comment by thisconnect** [comment]
@benma PTAL and can you do the locize pull?

**Review by benma — APPROVED** [review]

**Review by benma — APPROVED** [review]

**Pull request merged**